effect/jvm/src/main/scala/scalaz/effect/console.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2017 John A. De Goes. All rights reserved.
+package scalaz.effect
+
+package object console {
```

Comment:
should this be `package object`  or rather a normal `object`?

Reply:
What are the differences between them? I was thinking users would probably do:

```
import scalaz.effect.console._
```

Reply:
At the very least it is a convention that `package object console` goes into `console/package.scala`.

Reply:
Fixed. 👍